frontend/src/scenes/billing/Billing.tsx

```diff
@@ -1,27 +1,57 @@
 import React from 'react'
 import { PageHeader } from 'lib/components/PageHeader'
-import { CurrentPlan } from './CurrentPlan'
+import { Plan } from './Plan'
 import { CurrentUsage } from './CurrentUsage'
 import { BillingEnrollment } from './BillingEnrollment'
-import { useValues } from 'kea'
 import './Billing.scss'
 import { billingLogic } from './billingLogic'
 import { SceneExport } from 'scenes/sceneTypes'
+import { LemonButton, LemonDivider } from '@posthog/lemon-ui'
+import { AlertMessage } from 'lib/components/AlertMessage'
+import { useValues } from 'kea'
 
 export const scene: SceneExport = {
     component: Billing,
     logic: billingLogic,
 }
 
 export function Billing(): JSX.Element {
-    const { billing } = useValues(billingLogic)
+    const { billing, isSmallScreen } = useValues(billingLogic)
 
     return (
-        <div className="BillingPage">
-            <PageHeader title="Billing &amp; usage information" />
-            <CurrentUsage />
-            {billing?.plan ? <CurrentPlan plan={billing.plan} /> : <BillingEnrollment />}
-            <div style={{ marginBottom: 128 }} />
+        <div className="flex flex-col space-y-6">
+            <PageHeader title="Billing &amp; usage" />
+            <LemonDivider />
+            {billing?.should_setup_billing && (
+                <AlertMessage
+                    type="warning"
+                    action={{
+                        children: billing.subscription_url ? (
+                            <a href={billing.subscription_url}>
+                                <LemonButton>Finish setup</LemonButton>
+                            </a>
+                        ) : undefined,
+                    }}
+                >
+                    Your plan is <b>currently inactive</b> as you haven't finished setting up your billing information.
+                </AlertMessage>
+            )}
+            {isSmallScreen && (
+                <div className="flex flex-col space-y-4">
+                    <CurrentUsage />
+                    {billing?.plan ? <Plan plan={billing.plan} currentPlan /> : <BillingEnrollment />}
+                </div>
+            )}
+            {!isSmallScreen && (
```

Comment:
1. This could be a bit more obvious by using the ternary operator
2. Ideally we should be able to handle responsiveness purely in CSS with media queries, which are more efficient than re-rendering components. Though I see how this isn't doable purely with our Tailwind-lite